test/e2e/scenarios/Cluster/cluster_create_update.yaml

```diff
@@ -137,8 +134,7 @@ steps:
         conditions:
           ACK.ResourceSynced:
             status: "True"
-            timeout: 100
-
+            timeout: 180
```

Comment:
3 minutes to create a parameter group? My basic tests show that they are created practically instantly when using the command line. Do you know where the extra time is spent before it reaches `ResourceSynced`?

Reply:
Will create a new PR to fix all timeout.

Reply:
Set it back to 100 to keep it same, and will have a new PR for finding a more appropriate value.